code/processes/tickerlogreplay.q

```diff
@@ -4,6 +4,7 @@
 
 // Variables
 firstmessage:@[value;`firstmessage;0]			// the first message to execute
+autoreplay:@[value;`autoreplay;1b]                      // replay tplogs automatically set to 1b to be backward compatible.
```

Comment:
Align comment with the rest

Reply:
Other comments are aligned with tabs so no need.